test/ClangImporter/Inputs/many-imports/module.modulemap

```diff
@@ -0,0 +1,100 @@
+module Module1 { header "module1.h" }
```

Comment:
Does this test require the modules to be non-empty?  If not, it would be nice to omit the headers.